config/plugins.js

```diff
@@ -1,5 +1,5 @@
 module.exports = ({ env }) => ({
-  upload: {
+  /*  upload: {
```

Comment:
I think we are still using the Azure upload plugin for other type of uploads though. For example, some images are uploaded to Azure.